### PR TITLE
Acl UI implementation and bug fixes

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Update is ACL implementation.
* Include 403 component.
* Handle TEST FUNCTION action based on permissions.

### Description of the Change

* Updated is ACL implementation.
* Included 403 component.
* Handled TEST FUNCTION action based on permissions.

### CODE REVIEW

File: core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.ts [hunk 1]
```diff
@@ -188,6 +187,9 @@ export class EnvironmentDetailComponent implements OnInit {
 
       },
       err => {
+        if(err.status === "403" || err.type === 3) {
```
Comment (SauarvD) [inline core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.ts]: status is number

File: core/jazz_ui/src/app/pages/error403/error403.component.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.error-txt-wrp {
+  position: absolute;
+  width: 100%;
+  bottom: 10%;
+}
+
+.error-txt {
+  padding-bottom: 25px;
+  font-size: 1.5rem;
+  // a {
```
Comment (SauarvD) [inline core/jazz_ui/src/app/pages/error403/error403.component.scss]: remove commented lines

File: core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.ts (hunk 1 above)
Comment (supritat) [reply core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.ts]: Updated

File: core/jazz_ui/src/app/pages/error403/error403.component.scss (hunk 2 above)
Comment (supritat) [reply core/jazz_ui/src/app/pages/error403/error403.component.scss]: Removed

**Pull request merged**